src/status_im/transport/core.cljs

```diff
@@ -0,0 +1,66 @@
+(ns status-im.transport.core
+  (:require [cljs.spec.alpha :as spec]
+            [re-frame.core :as re-frame]
+            [status-im.constants :as constants]
+            [status-im.transport.message.core :as message]
+            [status-im.transport.filters :as filters]
+            [status-im.transport.utils :as transport.utils]
+            [status-im.utils.config :as config]
+            [taoensso.timbre :as log]
+            status-im.transport.inbox
+            [status-im.utils.handlers :as handlers]
+            [status-im.transport.db :as transport.db]))
+
+(defn initialize-offline-inbox [_]
+  (when config/offline-inbox-enabled?
+    (log/debug :init-offline-inbox)
+    {:dispatch [:initialize-offline-inbox]}))
+
+(defn init-whisper
```

Comment:
Doc string elaborating on what it is doing (discovery filter/sym-key/offline-inboxing)